docs/data/sdks/marketing-analytics-browser/index.md

```diff
@@ -0,0 +1,100 @@
+
+amplitude.init("YOUR_API_KEY_HERE");
+</script>
+```
+
+## Usage
+
+### Initialize the SDK
+
+--8<-- "includes/sdk-httpv2-notice.md"
+
+--8<-- "includes/sdk-ts-browser/init.md"
+
+### Configuration
+
+#### Basic options
+
+--8<-- "includes/sdk-ts-browser/shared-configurations.md"
+
+#### Attribution and tracking options
+
+|<div class="big-column">Name  </div>    | Value|Description|
+|----|----|----|
+|`trackingOptions`| Optional. `TrackingOptions` | [Learn more about tracking options](https://www.docs.developers.amplitude.com/data/sdks/marketing-analytics-browser/#optional-tracking/). |
+|`attribution.disabled`| Optional. `boolean` | Disable the attribution tracking, attribution is enabled by default |
+|`attribution.excludeReferrers`|  Optional. `string[]` | Exclude the attribution tracking for the provided referrers string |
+|`attribution.initialEmptyValue`| Optional. `string` | Reset the `sessionId` on a new campaign, Default value is `EMPTY` |
+|`attribution.resetSessionOnNewCampaign`| Optional. `boolean` | Reset the `sessionId` on a new campaign, won't create a new session for new campaign by default |
+|`pageViewTracking.trackOn`| Optional. `attribution` or `() => boolean` | `attribution` - Fire a page view event attribution information changes. `undefined` - Fire a page view event on page load or on history changes for single page application, default behavior. `() => boolean` - Fire a page view events based on a `trackOn` functions|
+|`attribution.pageViewTracking.trackHistoryChanges`  | Optional. `pathOnly` or `all` | Track the page view only on the path changes, track `all` URL changes by default|
+
+--8<-- "includes/sdk-ts-browser/basic-ts-functions.md"
+
+--8<-- "includes/sdk-ts-browser/marketing-analystic.md"
```

Comment:
is "analystic" the right term? If this is a typo, we should fix the filename before we ship.